website/templates/docs.html

```diff
@@ -409,7 +410,7 @@ <h2 id=for>For loop</h2>
 because with the latter it's easy to forget to update the counter and get
 stuck in an infinite loop.
 
-
+TODO: Break and continue.
```

Comment:
Lots of things are missing, I don't wont to pollute the docs with TODO

Reply:
Fair enough ;-)